### ISSUE TITLE
Last run output window empty in Windows 10

### ISSUE THREAD

**Issue by ikjordan** [SimonGreenaway/sQLux-menu#28]
When using sQLux-menu under Windows 10 and launching sqlux.exe no data is added to the "Last run output" window. 
Instead it goes into the command window that was used to launch  sQLux-menu

**Comment by xXorAa** [comment]
That is almost certainly the fault of this patch in sQLux

https://github.com/SinclairQL/sQLux/commit/cec6b0c091e548e01112ac5081b22336ad6ce727

**Comment by ikjordan** [comment]
I think you are right. Unfortunately without that patch I do not think there is a way of seeing the sQLux output in Windows in the command window when starting sQLux from the command prompt. That is why I added it :-) The issue seems to be that when launched from sQLux-menu it looks to Windows that sQLux has been launched from the command line, rather than the GUI.

We may just have to accept that the Last run output cannot be captured. The options are either close this no change, or alternatively if feeling keen remove the last run output window from sQLux-menu when running under Windows.

**Comment by SimonGreenaway** [comment]
I think you are right, disable that under window as there is no way to capture the output.

**Comment by ikjordan** [comment]
Now I think about it more, could we get this to work by adding a command line argument to sQLux which disables the redirect behaviour that I added to sQLux? sQLux-menu could then always pass this command line argument when launching sQLux. It would have no effect for non Windows versions, but would hopefully allow the capture of the output in sQLux-menu.
I'll experiment today.

**Comment by SimonGreenaway** [comment]
Via an environment variable?

**Comment by ikjordan** [comment]
That would work if the variable was set by sQLux-menu prior to launching sQLux. 

I prototyped this by changing the line in https://github.com/SinclairQL/sQLux/commit/cec6b0c091e548e01112ac5081b22336ad6ce727 where `reattach_console` is called, to only execute if environment variable `SQLUX_WIN_DISABLE_CONSOLE_OUTPUT` was set. I then created a simple batch file:

`set SQLUX_WIN_DISABLE_CONSOLE_OUTPUT="ON"`
`java -jar sqluxMenu.jar -s ./sqlux.exe`
`set SQLUX_WIN_DISABLE_CONSOLE_OUTPUT=`

Executing the batch file allowed the output to be displayed (after sQLux terminated), whilst still allowing output when sqlux was launched from the command line.

If we think this a way forward then I can raise a pull request for sQLux. I think the use of a batch file would then need to be documented in this project.
